nan.h

```diff
@@ -1847,7 +1847,7 @@ NAN_INLINE v8::Local<v8::Value> NanEncode(
 #if (NODE_MODULE_VERSION > 0x000B)
   return node::Encode(
       v8::Isolate::GetCurrent()
-    , buf, len
+    , (const char *)buf, len
```

Comment:
Yes, but the cast is strictly necessary only for io.js.

But it might also be that NanEncode itself should take a const char * instead of a const void *. It would remove the need for casting and version checking, but would make less sense when using UCS2. Haven't decided on the best course of action.

On January 23, 2015 12:49:51 PM EET, Flandre Scarlet <notifications@github.com> wrote:
>> @@ -1847,7 +1847,7 @@ NAN_INLINE v8::Local<v8::Value> NanEncode(
>>  #if (NODE_MODULE_VERSION > 0x000B)
>>    return node::Encode(
>>        v8::Isolate::GetCurrent()
>> -    , buf, len
>> +    , (const char *)buf, len
>
>@kkoopa  you mean use `static_cast<const char *>(buf)` to instead
>`(const char *)buf`?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rvagg/nan/pull/265/files#r23443092